auto-generated/cluster/stone-prd-rh01/tenants/rhtap-o11y-tenant/appstudio.redhat.com_v1beta1_integrationtestscenario_tools-enterprise-contract.yaml

```diff
@@ -0,0 +1,22 @@
+apiVersion: appstudio.redhat.com/v1beta1
+kind: IntegrationTestScenario
+metadata:
+  name: tools-enterprise-contract
+  namespace: rhtap-o11y-tenant
+spec:
+  application: tools
+  contexts:
+  - description: Application testing
+    name: application
+  params:
+  - name: POLICY_CONFIGURATION
+    value: rh-managed-rhtap-ser-tenant/rh-policy-o11y
```

Comment:
Make sure the policy matches with the RPA, so we now from the Integration Test if it is going to pass or not

Reply:
I know we had some issues with that, and I'm not sure why `o11y` is still using different policies for integration and release plan.

Reply:
It doesn't make a lot of sense to have them not match. Since the integration test what is does is giving you a heads-up of what the release pipeline is going to test. So if we use this one here and the other (which is stricter) in the release, the test will pass and the release will fail

Reply:
I don't see `rh-managed-rhtap-ser-tenant/rh-policy` being mentioned anywhere. Is that the default for `POLICY_CONFIGURATION`?

Reply:
I see the policy here: https://github.com/redhat-appstudio/tenants-config/blob/main/auto-generated/cluster/stone-prd-rh01/managed/rh-managed-rhtap-ser-tenant/appstudio.redhat.com_v1alpha1_enterprisecontractpolicy_rh-policy.yaml 

There are some exclusions, so it makes sense what you are currently doing. If you don't put anything you will use the redhat-no-hermetic by default